app/controllers/api/dishes_controller.rb

```diff
@@ -26,7 +26,8 @@ def update
       dish = find_dish
       authorize dish
       update_record(dish, dish_params) do |this_dish|
-        if current_user.id == this_dish.user_dishes.find_by(dish_owner: true).user_id
+        if current_user.id == this_dish.user_dishes
+            .find_by(dish_owner: true).user_id
```

Comment:
nit:
I find this line break confusing. If it's not fitting in 80 chars I would extract `this_dish.user_dishes.find_by(dish_owner: true).user_id` to a variable